src/coinm-client.ts

```diff
@@ -116,116 +116,7 @@ export class CoinMClient extends BaseRestClient {
     );
   }
 
-  /**
-   *
-   * Broker Futures Endpoints
-   *
-   **/
-
-  // 1 == USDT-Margined, 2 == Coin-margined
-  getBrokerIfNewFuturesUser(
```

Comment:
All broker stuff, as marked in all clients, now seems to be /broker/ - you can read more in other files.